File: 프로그래머스/unrated/181887. 홀수 vs 짝수/홀수 vs 짝수.py
def solution(num_list):
    a = sum([ num_list[i] for i in range(0, len(num_list), 2)])
    b = sum([ num_list[i] for i in range(1, len(num_list), 2)])
    print(a, b)
    return a if a > b else b
        

File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.java
class Solution {
    public int solution(int[] num_list) {
        int evenSum = 0;
        int oddSum = 0;
        for (int i = 0; i<num_list.length; i+=2) {
            evenSum += num_list[i];
        }
        for (int i = 1; i<num_list.length; i+=2) {
            oddSum += num_list[i];
        }
        return Math.max(evenSum, oddSum);
    }
}